openapi/components/schemas/WorkflowEngineVersion.yaml

```diff
@@ -0,0 +1,9 @@
+title: WorkflowEngineVersion
+type: object
+properties:
+  workflow_engine_version:
+    type: array
+    items:
+      type: string
+    description: An array of one or more acceptable engines for the `workflow_engine`
```

Comment:
I think this description is more so for `workflow_engine` and not for `workflow_engine_version`?

Reply:
@patmagee updated.